iothub_client/samples/iothub_ll_c2d_sample/linux/CMakeLists.txt

```diff
@@ -1,20 +1,22 @@
 #Copyright (c) Microsoft. All rights reserved.
 #Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#this is CMakeLists.txt for iothub_ll_c2d_sample
-cmake_minimum_required(VERSION 2.8.11)
+#this is CMakeLists.txt for iothub_ll_telemetry_sample
+cmake_minimum_required(VERSION 3.5)
+
+project(iothub_ll_telemetry_sample)
```

Comment:
This is actually converting your c2d sample into the telemetry sample -- sorry for the shortcut I took here!